app/services/record_declaration.rb

```diff
@@ -286,13 +285,4 @@ def create_participant_outcome!
   def check_mentor_completion
     ParticipantDeclarations::HandleMentorCompletion.call(participant_declaration:)
   end
-
-  def validate_if_npq_course_supported
```

Comment:
Do we not want to keep this? 🤔

Reply:
agree to keep

Reply:
Yes, will change this, i saw it on Leandro's PR👍